code/modules/power/supermatter/supermatter.dm

```diff
@@ -510,7 +521,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			src.fire_nuclear_particle()        // Start to emit radballs at a maximum of 30% chance per tick
 
 		//Power * 0.55
-		var/device_energy = power * REACTION_POWER_MODIFIER
+		var/device_energy = power * REACTION_POWER_MODIFIER * (psyCoeff == 1 ? 0.5 : 1)
```

Comment:
It seems like everywhere psyCoeff affects at a rate of "0.5". Why not make psyCoeff maximally go to 0.5(and also make a define like MAX_PSYCOEFF), but to mitigate the speed of getting to max change the change rate from 0.05 to 0.025?

Reply:
I'd prefer to keep it between 0 and 1 for understandability.